### ISSUE TITLE
Auto-save prevents Animation::TransitionTime to be saved on MacOS

### ISSUE THREAD

**Issue by AntonCha** [SketchUp/api-issue-tracker#846]
1. SketchUp/LayOut Version: SketchUp Pro 2022 / Version:  22.0.353
2. OS Platform: MacOS M1

When checking the TransitionTime, set from Animation > Settings inside SketchUp, the ruby code below fetches the correct values, but if Preferences > General > Auto-save is enabled , once the auto save occurs, changing the TransitionTime value from SketchUp does not affect the returned result from the ruby code. 

This issue is only reproducible on Mac, on Windows it seems to work correctly even after auto-save occurs.

```ruby
model = Sketchup.active_model
manager = model.options
manager.each { | p |
    next unless p.name == "PageOptions"
    p.each_pair{ | k, v | 
        next unless k == "TransitionTime"
        puts k
        puts v
    }
}
```

**Comment by thomthom** [comment]
Have you had a chance to test older versions?

**Comment by AntonCha** [comment]
No, i have not. Its the current version of SketchUp i have, but the issue might also be reproducible in older versions.

**Comment by DanRathbun** [comment]
Why mess around with iterations ?
```ruby
model = Sketchup.active_model
trans = model.options["PageOptions"]["TransitionTime"]
```

**Comment by AntonCha** [comment]
i fail to see the relevance of you question to the bug i have found and the possible information you would need to reproduce it and fix it.

**Comment by DanRathbun** [comment]
Occum would say, "_The simplest reproducible code snippet is the best_."
